app/router.js

```diff
@@ -65,9 +65,13 @@ Router.map(function() {
 
   this.route('project', { path: '/:slugged_route_slug/:project_slug' }, function() {
     this.route('checkout'); // Where you enter your credit card details
+    this.route('contributors');
```

Comment:
`contributors` was under `settings`, meaning, since `settings` is owner only, `contributors` was also owner only. To make it `admin` or `owner`, I had to move it.

If we move it back, each settings subroute will have to implement it's own `CanMixin`, while the base `settings` route should not have it.

If we want it here, but keep the old `/:orgslug/slug/settings/contributors` url, we can just set the path here.

Personally, I prefer the new placement and the new `:orgslug/:slug/contributors` url. I would even move `donations` in the same way

Reply:
I agree with this move.